### ISSUE TITLE
Breaking: No longer passing command result / error as named parameters to callbacks

### ISSUE THREAD

**Issue by pavsaund** [dolittle-interaction/JavaScript.Client.Aurelia#18]
Looks like the expected behaviour from #6 has been changed during the upgrade to TypeScript.
Was this intentional?

**Comment by woksin** [comment]
Yes, but can be changed back. Is it preferred havi g named params? As I remember that they only had one parmeter any way

**Comment by woksin** [comment]
If it causes issues then we can just change it back and call it a patch

**Comment by pavsaund** [comment]
I thought the original #6 had the link to the docs but doesn't look like it. Here's some more information from the official docs: https://aurelia.io/docs/binding/basics#function-references
